c2corg_ui/static/js/documentediting.js

```diff
@@ -354,5 +382,204 @@ app.DocumentEditingController.prototype.getCoordinatesFromPoint_ = function(
 };
 
 
+/**
+ * Clone input
+ * @export
+ */
+
+app.DocumentEditingController.prototype.addAnotherMapsInfo = function() {
+  var template = $('#maps_info-group input:last-of-type').val('').clone();
+  template.insertBefore($('#document-add-maps'));
+}
+
+
+/**
+ * Navigate through creation/editing steps
+ * @param {number} step
+ * @param {string} direction
+ * @export
+ */
+
+app.DocumentEditingController.prototype.step = function(step, direction) {
```

Comment:
Right. The wireframes only suggest 4 steps for the outings. So I guess 5 steps is enough anyway.